vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -118,17 +118,17 @@ struct ReactDevOptions {
   std::string DebugHost;
 
   //! When using web debugging and/or live reload, the source is obtained from the packager.
-  //! The source url for the bundle is "http://{HOST}:{PORT}/{PATH}{EXTENSION}?platform=..."
+  //! The source url for the bundle is "http://{HOST}:{PORT}/{NAME}{EXTENSION}?platform=..."
   //! which defaults to:
   //! {HOST} = "localhost",
   //! {PORT} = "8081",
-  //! {PATH} = "index.{PLATFORM}" where {PLATFORM} = "ios", "android", "win32", etc.
+  //! {NAME} = "index.{PLATFORM}" where {PLATFORM} = "ios", "android", "win32", etc.
   //! {EXTENSION} = ".bundle"
   //! Specify a value for a component, or leave empty to use the default.
-  std::string SourceBundleHost; // {HOST}
-  std::string SourceBundlePort; // {PORT}
-  std::string SourceBundlePath; // {PATH}
-  std::string SourceBundleExtension; // {EXTENSION}
+  std::string SourceBundleHost;      // Host domain (without port) for the bundler server. Default: "localhost".
+  std::string SourceBundlePort;      // Host port for the bundler server. Default: "8081".
```

Comment:
>  std::string SourceBundlePort [](start = 0, length = 30)

why not just store this as a ushort?

Reply:
Eventually all these stuff will be assembled to a URL for debug that's then passed to React Native as a std::string. I think storing `ushort` might not be worth the trouble of `itoa`?